content/developer/reference/ui/scss_optimization.rst

```diff
@@ -0,0 +1,203 @@
+=========
+SCSS Tips
+=========
+
+There are many ways to lean and simplify SCSS.
+The first step is to ensablish if custom code is needed at all.
+
+Odoo's webclient has been designed to be modular, meaning that (potentially all) classes can be shared across views.
+Check the code before creating a new class. Chances are that there is already a class or a tag doing exaclty what you're looking for.
+
+
+On top of that, odoo relies on `Bootstrap <https://getbootstrap.com/docs/5.1/getting-started/introduction/>`_ (BS), one of the most complete CSS framework available. The framework has been customised in order to match odoo design (both community and enterprise versions), meaning that you can use any BS class, direclty in odoo and achieve a visual result consistent with our UI.
+
+.. danger::
```

Comment:
Not sure i would use danger alert message, it feels a bit aggressive here. I guess warning could do the job ? But it's kinda subjective